pkg/util/ipvs/testing/fake.go

```diff
@@ -55,20 +55,20 @@ func toServiceKey(serv *utilipvs.VirtualServer) serviceKey {
 	}
 }
 
-//EnsureVirtualServerAddressBind is a fake implementation
+//EnsureVirtualServerAddressBind is an empty implementation
```

Comment:
No need to modify the comment here?

Reply:
Glad to receive your feedback, @dixudx 

I wrote the original comment, and I find it can be improved now. I just fix it conveniently - I don't want to raise another PR...